templates/compliance/components/UnreportedSystemsTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Th,
+  Tr,
+  Td,
+} from '@patternfly/react-table';
+
+export interface UnreportedSystemsProps {
```

Comment:
This interface definition looks very similar to the `SupportedSystemsProps` defined above. Can we have just one? And if there are differences we can definitely create a common type for some of the inner parts.